tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -119,14 +120,16 @@ stages:
             runOnce: 
               deploy:
                 steps: 
-                  - checkout: none
+                  - checkout: azure-sdk-build-tools
+                    path: azure-sdk-build-tools
                   - download: current
                     artifact: release-artifacts
+                  - template: pipelines/steps/binary-signing.yml@azure-sdk-build-tools
+                    parameters:
+                      BinaryPath: $(Pipeline.Workspace)/release-artifacts
                   - pwsh: |
-                      $latestVersion = (azcopy list "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/") -replace '[\/;].*' | Get-Unique | sort {[version] $_} -Descending | Select-Object -First 1
-                      $major, $minor, $build = $latestVersion.Split('.')
-                      $nextVersion = [version]::new($major, 1 + $minor, $build).ToString()
-                      Write-Host "##vso[task.setvariable variable=APIVIEW_PARSER_VERSION]$nextVersion"
+                      Write-Host "Set Api Parser version to $(OfficialBuildId)"
```

Comment:
Why not just use that variable inline below instead of setting another variable in this step? I suspect we can just remove this step.